osf/management/commands/force_archive.py

```diff
@@ -337,7 +361,18 @@ def _recurse(file_obj, node):
             'version': int(file_obj.versions.latest('created').identifier) if file_obj.versions.exists() else None
         }
         if not file_obj.is_file:
-            serialized['children'] = [_recurse(child, node) for child in OsfStorageFileNode.objects.filter(target_object_id=node.id, target_content_type_id=ct_id, parent_id=file_obj.id)]
+            active_children = list(OsfStorageFileNode.objects.filter(
+                target_object_id=node.id,
+                target_content_type_id=ct_id,
+                parent_id=file_obj.id
+            ))
+            trashed_children = list(TrashedFileNode.objects.filter(
+                target_object_id=node.id,
+                target_content_type_id=ct_id,
+                parent_id=file_obj.id
+            ))
```

Comment:
I think this makes sense. It looks like you're ensuring that if someone selects a node for archiving, then deletes a child, that the child is still included in the archiving. What happens if the child is deleted before it's selected for archiving? Is it still included in the archive? It should not be.

If possible, please make a couple of tests for these situations (or locate existing tests if they exist).